packages/cache/package.json

```diff
@@ -43,11 +43,11 @@
     "ci:eslint": "eslint \"**/*.{js,jsx,cjs,mjs,ts,tsx,cts,mts}\"",
     "ci:publint": "publint --strict",
     "ci:test": "vitest run --coverage --typecheck",
-    "ci:test:watch": "vitest --ui --coverage --typecheck",
     "ci:type": "tsc --noEmit",
     "clean": "rimraf ./dist && rimraf ./coverage",
     "dev": "tsup --watch",
```

Comment:
```suggestion
```

Reply:
Since watch for each package is not necessary and the build:watch in the root directory takes care of that, I think it's fine not to have it.